src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,6 +1,9 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.Email;
+
 public class UserRequestDto {
+    @Email
```

Comment:
Implement custom email validation.
you created custom annotation, let's use it here